now.json

```diff
@@ -0,0 +1,19 @@
+{
+  "redirects": [
+    { "source": "/about\\.html", "destination": "/about" },
+    { "source": "/foundation/", "destination": "/about#foundation" },
+    { "source": "/team\\.html", "destination": "/about" },
+    { "source": "/press\\.html", "destination": "/about" },
+    { "source": "/contact\\.html", "destination": "/#get-in-touch" },
+    { "source": "/sponsors\\.html", "destination": "/contribute#supporter" },
+    { "source": "/imprint\\.html", "destination": "/imprint" },
+    { "source": "/material\\.html", "destination": "https://learn.opentechschool.org/" },
+    { "source": "/berlin/", "destination": "/cities/berlin" },
+    { "source": "/leipzig/", "destination": "/cities/leipzig" },
+    { "source": "/zurich/", "destination": "/cities/zurich" },
+    { "source": "/brussels/", "destination": "/cities/brussels" },
+    { "source": "/bristol/", "destination": "/cities/bristol" },
+    { "source": "/romagna/", "destination": "/cities/romagna" },
+    { "source": "/handbooks/(.*)", "destination": "/guides" }
```

Comment:
Does this work for urls with multiple paths /*/*/*?

Reply:
Yes.